build.sh

```diff
@@ -362,11 +366,19 @@ fi
 
 # Disable the native build when targeting Windows.
 
-if [ "$__BuildOS" != "$__HostOS" ]; then
+if [ "$__BuildOS" != "$__HostOS" ] && [ $__CrossBuild == 0 ]; then
```

Comment:
I'm confused by this part.  We warn that cross-compilation isn't yet supported if CrossBuild isn't set, but if it is set, we don't warn?